docs/source/changelog.rst

```diff
@@ -14,11 +14,13 @@ Changelog
 **v2.0.0** Oct 13, 2021
     * Changes
         * Update primitives for compatibility with Featuretools 1.0.0 (:pr:`61`)
+        * Update setup.py with new and correct information (:pr:`72)
```

Comment:
this should be in the future release section

Reply:
My bad. Fixed